### ISSUE TITLE
Is this the official React adapter to MCW?

### ISSUE THREAD

**Issue by jimbojetlag** [material-components/material-components-web-react#26]
[Readme file of MCW](https://github.com/material-components/material-components-web/blob/master/README.md) does not point to this project as the React adapter. Is this official? Will it be maintained, and is it safe to base production projects on it?

**Comment by abraham** [comment]
This was announced at I/O.

https://events.google.com/io/schedule/?section=day&sid=49ccebfb-2250-4f0a-86df-dbb5c3f0a594

**Comment by moog16** [comment]
@abraham yes this is the official Material Design React Wrapper of the [MDC Web](https://github.com/material-components/material-components-web). We are in alpha similar to MDC Web, and will continue maintain the project. We will try to keep it as up to date with the latest version of React and latest version of MDC Web. Currently an issue open to upgrade to v0.35.0.
